datarepo-clienttests/src/main/resources/suites/StagingSmokeTests.json

```diff
@@ -2,5 +2,12 @@
   "name": "StagingSmokeTests",
   "description": "Smoke tests to run in staging environment.",
   "serverSpecificationFile": "staging.json",
-  "testConfigurationFiles": ["functional/UserWorkflow.json"]
+  "testConfigurationFiles": [
+    "functional/UserWorkflow.json",
```

Comment:
This test creates and retrieves a snapshot, then does a DRS lookup: https://github.com/DataBiosphere/jade-data-repo/blob/develop/datarepo-clienttests/src/main/java/scripts/testscripts/RetrieveSnapshot.java

I'm thinking we could remove this test and add the snapshot retrieval and DRS lookup to `CreateSnapshot.json` and `AzureCreateSnapshot.json` to consolidate this a bit more 🤔